app/controllers/orgs/rosters_controller.rb

```diff
@@ -107,6 +109,14 @@ def add_student
       redirect_to roster_path(current_organization)
     end
 
+    def download_roster
+      respond_to do |format|
+        format.csv { send_data @roster_entries.to_csv, filename: "classroom_roster.csv", disposition: "attachment" }
+      end
+
+      return
```

Comment:
You can yank this `return` statement, it doesn't do anything here

Reply:
I did it so we won't have roster.csv in the url but it's the same without it indeed, thanks !